st2actions/st2actions/runners/python_action_wrapper.py

```diff
@@ -62,6 +61,8 @@ def __init__(self, pack, file_path, parameters=None, parent_args=None):
             config.parse_args(args=self._parent_args)
         except Exception:
             pass
+        else:
+            db_setup()
```

Comment:
The original reason, I believe, why I wrapped it in try / except is that in some cases, config is already parses at that point (don't remember where and why) and if you try to parse it again it will throw.

Obviously this try / except is not the best (ideally we would have a consistent place where a config is parsed and we would never try to do it twice), but our config parsing and bootstrap code is less than ideal so that's not always that easy to do.